app/addons/activetasks/components.react.jsx

```diff
@@ -0,0 +1,610 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'react',
+  'addons/activetasks/stores',
+  'addons/activetasks/resources',
+  'addons/activetasks/actions'
+], function (app, FauxtonAPI, React, Stores, Resources, Actions) {
+
+  var activeTasksStore = Stores.activeTasksStore;
+  var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
+
+  var ActiveTasksController = React.createClass({
+
+    getStoreState: function () {
+      return {
+        collection: activeTasksStore.getCollection(),
+        searchTerm: activeTasksStore.getSearchTerm(),
+        selectedRadio: activeTasksStore.getSelectedRadio(),
+
+        sortByHeader: activeTasksStore.getSortByHeader(),
+        headerIsAscending: activeTasksStore.getHeaderIsAscending(),
+
+        setPolling: activeTasksStore.setPolling,
+        clearPolling: activeTasksStore.clearPolling,
+      };
+    },
+
+    getInitialState: function () {
+      return this.getStoreState();
+    },
+
+    componentDidMount: function () {
+      this.state.setPolling();
+      activeTasksStore.on('change', this.onChange, this);
+    },
+
+    componentWillUnmount: function () {
+      this.state.clearPolling();
+      activeTasksStore.off('change', this.onChange, this);
+    },
+
+    onChange: function () {
+      this.setState(this.getStoreState());
+    },
+
+    setNewSearchTerm: function (searchTerm) {
+      Actions.setSearchTerm(searchTerm);
+    },
+
+    switchTab: function (newRadioButton) {  //radio buttons
+      Actions.switchTab(newRadioButton);
+    },
+
+    tableHeaderOnClick: function (headerClicked) {
+      Actions.sortByColumnHeader(headerClicked);
+    },
+
+    render: function () {
+      var collection = this.state.collection;
+      var searchTerm = this.state.searchTerm;
+      var selectedRadio = this.state.selectedRadio;
+      var sortByHeader = this.state.sortByHeader;
+      var headerIsAscending = this.state.headerIsAscending;
+
+      var setSearchTerm = this.setNewSearchTerm;
+      var onTableHeaderClick = this.tableHeaderOnClick;
+
+      if (collection.length === 0 ) {
+        return (<div className="active-tasks"><tr><td><p>  No active tasks. </p></td></tr></div>);
+      } else {
```

Comment:
no need for an else as you have an "early return" above.

if collection.length === 0 the code beneath is not executed.